app/assets/stylesheets/scaffolds.scss

```diff
@@ -162,13 +162,12 @@ body.static-navbar-config {
   }
 }
 
-.pro-visible-block,
 .trusted-visible-block {
   display: none !important;
 }
 
 body.trusted-status-true .trusted-visible-block {
-  display: block !important;
+  display: flex !important;
```

Comment:
We use this weird "CSS check" to show/hide things for moderators.. But this feels like something that should be done in backend rather than frontend, am I right?

Also, it's there ONLY for one link.

Reply:
It should be, but from what I remember when I asked about this is even if a non-moderator hacked the frontend, the backend would still reject it.

Reply:
hmm that makes me wonder even more more why we can't just check it in backend?

not like it's make a huge difference right now but i'd love to unclutter our frontend code as much as possible.

Reply:
Let's leave it for another PR and/or discuss with the team, but I agree.